server/config/default/kustomization.yaml

```diff
@@ -24,15 +23,15 @@ replacements:
     targets:
       # create Role and RoleBinding to read SpaceBinding into toolchain-host-operator
       - options:
-          create: false
+          create: true
```

Comment:
Out of curiosity - why is this change?

Reply:
Reverted in 8aeec18.

Reply:
It's needed since kustomize fails: https://github.com/konflux-workspaces/workspaces/actions/runs/9663771194/job/26656780312?pr=214

Reply:
ok